docker-compose.yaml

```diff
@@ -9,7 +9,7 @@ services:
       - '3306:3306'
     labels:
       lagoon.type: mariadb-single
-      lagoon.image: amazeeiolagoon/api-db:v1-1-4
+      lagoon.image: amazeeiolagoon/api-db:pr-1269
```

Comment:
please leave these with `v1-1-4` or master will not be able to deploy anymore.
also there is a merge conflict :)

Reply:
Updated as requested.